react/react2/react2-week1-ex2/app/page.js

```diff
@@ -0,0 +1,12 @@
+import Image from "next/image";
+import styles from "./page.module.css";
+import Navbar from "./components/Navbar";
+
+export default function Home() {
+  return (
+    <div>
+      {" "}
+      <Navbar />
```

Comment:
I think it might be not very useful to have a navbar that disappears every time you use it :) I think there is might be a better place to put it, that'll affect not only main page, but also all ancestors pages (= put in `layout.js`)